src/v/cloud_storage/read_path_probes.cc

```diff
@@ -63,90 +63,107 @@ partition_probe::partition_probe(const model::ntp& ntp) {
 }
 
 ts_read_path_probe::ts_read_path_probe() {
-    if (config::shard_local_cfg().disable_metrics()) {
-        return;
-    }
-
     namespace sm = ss::metrics;
     auto aggregate_labels = std::vector<sm::label>{sm::shard_label};
 
-    _metrics.add_group(
```

Comment:
>cloud_storage: Add public throttling metric

in the commit message, can you indicate what metrics were made public, and what metrics, if any, changed.

Reply:
done